Reorganize "edit" docs (#3456)
DataDog/system-tests: docs/edit/features.md

> System tests are feature-oriented; put another way, tests certify which tracing libraries (and on what versions) support which features. Each test class must belong to a "feature," where "features" map to entries in the [Feature Parity Dashbaord](https://feature-parity.us1.prod.dog/). We use the @features decorators to achieve this.

Reviewer: system tests are also used for non tracing products we should be clear about that
```suggestion
System tests are feature-oriented; put another way, tests certify which features are supported in each client library (and the supported library versions). Each test class must belong to a "feature", where "features" map to entries in the [Feature Parity Dashbaord](https://feature-parity.us1.prod.dog/). We use the @features decorators to achieve this.
```
